api/tests.py

```diff
@@ -366,6 +368,38 @@ def test_add_existing_test_jobs_to_existing_result_object(self):
         self.assertEqual(models.Manifest.objects.count(), 1)
         self.assertEqual(models.TestJob.objects.count(), 2)
 
+    @patch.dict('django.conf.settings.CREDENTIALS', {'jenkins.linaro.org': ("hej", "ho")})
+    def test_post_already_with_testjob_results(self):
+        data = {
+            'build_url': 'http://jenkins.linaro.org/foo/bar/baz/1',
+            'name': u'linaro-art-stable-m-build-juno',
+            'url': u'http://dynamicfixture1.com',
+            'build_number': 200,
+            'build_id': 20,
```

Comment:
I think build_number should be the same as build_id. I'm not sure why there are 2 fields for this property.